gradle-jdks/src/main/java/com/palantir/gradle/jdks/JdkManager.java

```diff
@@ -72,8 +76,14 @@ public Path jdk(JdkSpec jdkSpec) {
                 copy.into(temporaryJdkPath);
             });
 
+            Path javaHome = findJavaHome(temporaryJdkPath);
+
+            jdkSpec.caCerts().caCerts().forEach((name, caCertFile) -> {
+                addCaCert(javaHome, name, caCertFile);
+            });
```

Comment:
Hmm, given two projects which use azul-17 x86_64, what happens if both projects specify completely different certificates? Do we need to include the certificates in our cache key? I'm not sure that we should mutate the jvm in a potentially leaky way such that tests pass on project B only if you've run project A first.

Reply:
I made the CA certificates configurable at the root level/all jdks level, so all the jdks will have the same CA certs.

Reply:
Ah fantastic, I missed that in the block above :-)